coral/src/app/features/components/filters/TopicTypeFilter.tsx

```diff
@@ -0,0 +1,42 @@
+import { NativeSelect, Option } from "@aivenio/aquarium";
+import { useFiltersContext } from "src/app/features/components/filters/useFiltersContext";
+import { TopicType } from "src/domain/topic";
+import upperFirst from "lodash/upperFirst";
+
+type TopicTypeForFilter = TopicType | "ALL";
+const topicTypesForFilter: TopicTypeForFilter[] = [
+  "ALL",
+  "Consumer",
+  "Producer",
+];
+function TopicTypeFilter() {
+  const { topicType, setFilterValue } = useFiltersContext();
+
+  return (
+    <NativeSelect
+      labelText={"Filter by Topic type"}
+      defaultValue={topicType}
+      onChange={(e) => {
+        const selectedType = e.target.value as TopicTypeForFilter;
+        return setFilterValue({ name: "topicType", value: selectedType });
```

Comment:
This will currently set the param with the value uppercased, is it what we want, or should it be lowercase? :o 
Currently `topicType=Consumer`, should it be `topicType=consumer`?

Or do we not care? Haha!

Reply:
Ah, yes, good point. Currently the backend takes it with uppercase, that's why I just added it. But I'll align it with the other queries when the api is updated and the query is an enum!